packages/wallet-sdk/package.json

```diff
@@ -36,7 +36,6 @@
     "@solana/web3.js": "^1.78.0",
     "bind-decorator": "^1.0.11",
     "bn.js": "^5.2.1",
-    "buffer": "^6.0.3",
```

Comment:
Buffer doesn't exist in the browser. We'll be keeping this package and updating the code to reference it directly vs using the node builtin

Reply:
All current bundlers should do it by default now, but we can leave since you prefer it :+1: